pkg/issuer/acme/setup.go

```diff
@@ -21,6 +22,7 @@ import (
 const (
 	errorAccountRegistrationFailed = "ErrRegisterACMEAccount"
 	errorAccountVerificationFailed = "ErrVerifyACMEAccount"
+	errorDurationInvalid           = "ErrACMEDurationInvalid"
```

Comment:
Needn't be an ACME specific error. `ErrDurationInvalid` should suffice, and can be reused across issuers.